src/worker/ingestion/process-event.ts

```diff
@@ -288,6 +289,11 @@ export class EventsProcessor {
         const oldPerson = await this.db.fetchPerson(teamId, previousDistinctId)
         const newPerson = await this.db.fetchPerson(teamId, distinctId)
 
+        if (oldPerson?.is_identified) {
+            console.log('Person associated with previousDistinctId is identified, refusing to alias new distinctId')
```

Comment:
Include distinct ids in here as well. You don't need to use interpolation, just add `{ distinctId, previousDistinctId }` as the last argument